test_vm/bootstrap.sh

```diff
@@ -0,0 +1,8 @@
+# Create new physical volume.
+pvcreate /dev/sdb
+# Add the physical volume to the existing volume group.
+vgextend centos /dev/sdb
+# Add the physical volume to the logical volume.
+lvextend /dev/centos/root /dev/sdb
+# Resize the file system to use the new space available in the logical volume..
+xfs_growfs /dev/centos/root
```

Comment:
Is the file system indeed xfs (I think the default is ext3 or ext4)? Can we check the output of `mount` and `cat /etc/fstab`?